src/Symfony/Component/DomCrawler/Field/FileFormField.php

```diff
@@ -48,7 +48,7 @@ public function upload($value)
     /**
      * Sets the value of the field.
      *
-     * @param string $value The value of the field
+     * @param string|null $value The value of the field
```

Comment:
if the parent accepts a wider range of arguments, so should the child method, isn't it?
shouldn't we replace the whole block with `@inheritdoc`?